Documentation/ceph-dashboard.md

```diff
@@ -129,6 +129,30 @@ In this example, port `31176` will be opened to expose port `8443` from the ceph
 of the VM. If using minikube, you can run `minikube ip` to find the ip address.
 Now you can enter the URL in your browser such as `https://192.168.99.110:31176` and the dashboard will appear.
 
+### Load Balancer
+
+If you have a cluster on a cloud provider that support load balancers,
+you can create a service that is provisioned with a public hostname.
+The yaml is the same as `dashboard-external-https.yaml` except for the following line:
+```yaml
+  type: LoadBalancer
+```
+
```

Comment:
For consistency with the documentation on the other types of services, how about adding this section so it's obvious which example filename to use? 

Now create the service:
```
$ kubectl create -f dashboard-loadbalancer-https.yaml
```

Reply:
I agree. I dropped the `-https` suffix because I think this method is not restricted to https.